mobx/lib/src/core/reaction.dart

```diff
@@ -91,7 +91,7 @@ class ReactionImpl with DebugCreationStack implements Reaction {
     }
```

Comment:
Also need to bump up version and add a changelog for this

Reply:
done